app/src/ui/lib/text-box.tsx

```diff
@@ -178,6 +178,21 @@ export class TextBox extends React.Component<ITextBoxProps, ITextBoxState> {
   }
 
   private onKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (
+      this.state.value !== '' &&
+      this.props.type === 'search' &&
+      event.key === 'Escape'
+    ) {
+      const value: string = ''
```

Comment:
I don’t think you need the `: string` here.

Reply:
I didn't either, I put it there to appease VSCode. I'll try restarting my editor to see if it still complains when I remove the type annotation.